README.md

```diff
@@ -0,0 +1,24 @@
+[![Build Status](https://travis-ci.org/RIOT-OS/applications.svg?branch=master)](https://travis-ci.org/RIOT-OS/applications)
+
+This repository provides some applications for the [RIOT operating system][riot-repo]. Some of them are just useful tools for
```

Comment:
lots of `some`s in here, I'd suggest to change the first one and state `provides more applications for the RIOT ...`

Reply:
Done.